src/components/DiseaseTable.tsx

```diff
@@ -37,6 +39,89 @@ const DiseaseTable = ({
     status,
 }: DiseaseTableProps) => {
     const inProgress = status?.toLowerCase() === "coming soon";
+
+    const useWindowWidth = () => {
+        // so the build doesn't fail
+        if (typeof window === "undefined") {
+            return 0;
+        }
+        const [width, SetWidth] = useState(window.innerWidth);
+
+        useEffect(() => {
+            const handleResize = () => {
+                SetWidth(window.innerWidth);
+            };
+            const debouncedHandleResize = debounce(handleResize, 200);
+            window.addEventListener("resize", debouncedHandleResize);
+            return () => {
+                window.removeEventListener("resize", debouncedHandleResize);
+            };
+        }, []);
+        return width;
+    };
+
+    const width = useWindowWidth();
+    const isTablet = width < 768;
+    const isMobile = width < 576;
```

Comment:
nit & not for this pr: extracting them into a constants file to make them easier to update and maintain?